atticmatic/backends/borg.py

```diff
@@ -13,6 +13,7 @@
         (
             option('encryption_passphrase', required=False),
             option('compression', required=False),
+            option('umask', int, required=False),
```

Comment:
I did not test `77` but instead something more common like `0700`. Thanks for the review. I agree that using a string for it is better to avoid such problems. Done.